src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-details/pool-details.component.html

```diff
@@ -19,22 +19,22 @@
       <a ngbNavLink
          i18n>Performance Details</a>
       <ng-template ngbNavContent>
-        <cd-grafana [grafanaPath]="'ceph-pool-detail?var-pool_name='+ selection['pool_name']"
+        <cd-grafana [grafanaPath]="grafanaPath"
```

Comment:
I only changed this to make sure that no cycle takes to long and that the same measures are taken throughout the template ;) I used the build in profiler from my browser and I tested with pools that were created on cephadm managed hosts. I didn't noticed it before as pools from a vstart cluster are accessed pretty much instantly. Through that it took some time to get the data wanted. In the end it was a lot faster (over 60% AFAIK) from my side.

From the issue description:

> For the switch from details tab to configuration tab takes around 1s and the other way around takes 2 seconds where the UI doesn't really response.
Through using CephADM pools as well there is a longer delay to wait on data, but the data is polled every 5 seconds anyway.
This seems to cause the problem that the pool that is build for details will update it's detail data because of the above polling.

Reply:
Seriously, let's not overcomplicate the [things that Angular tells us it does finely ](https://angular.io/guide/interpolation):
```suggestion
        <cd-grafana grafanaPath="ceph-pool-detail?var-pool_name={{ selection.pool_name }}"
```
There are still hundreds of functions calls in our HTML templates, so let's start for that (and other big wins: like onpush change detection, caching and change detection in observables & service ...), rather than small improvements like string interpolation.

Reply:
done